### PR TITLE
RF-790: Add conditional logic to transfers screen

### PR DESCRIPTION
**Pull request by RockinRonE**
[Ticket](https://gathering.atlassian.net/browse/RF-790?atlOrigin=eyJpIjoiN2Y3YmVlMTJhNDVlNDBiYjgzNmFmZTljMzY5NWRmOGYiLCJwIjoiaiJ9)

I left the testData file for you all to check out. Just replace "account" with it within the account type check. 
Screen Shot 2021-12-30 at 10 32 50 AM

### CODE REVIEW

**Review by ksalmon — APPROVED** [review]

**Review by KramerJeff — CHANGES_REQUESTED** [review]
Seeing the PR notification in Slack bothered me so I'm going to say request changes for the conflicts and because as far as I know, we're waiting on backend to implement some features still.

**Comment by ksalmon** [comment]
@RockinRonE This work is now unblocked from this PR
https://github.com/RizeFinance/prototype-middleware/pull/65

File: screens/InitTransferScreen.tsx [hunk 1]
```diff
@@ -247,23 +266,61 @@ export default function InitTransferScreen({ navigation }: InitTransferScreenPro
...
+              <Checkbox
+                onChange={(value) => setFieldValue('checked', value)}
+                checked={values.checked}
+              >
+                <BodySmall style={{ marginBottom: 10 }} fontWeight="semibold">
+                  By proceeding with this transfer, I am acknowledging that as a consumer I am aware
+                  that my brokerage account with YieldX:
+                </BodySmall>
+                <BodySmall fontWeight="semibold" style={styles.marginVertical10}>
+                  &#8226; is NOT insured by the FDIC
```
Comment (KramerJeff) [inline screens/InitTransferScreen.tsx:304]: Why are we making bullets like this? Shouldn't we be using CSS or a UL element?

Comment (RockinRonE) [reply screens/InitTransferScreen.tsx:304]: @KramerJeff I know, I had the same question. Seems like RN doesn't do bullets. 

"React Native does not provide <ol> and <ul> like tags which auto handle bullet"

**Review by KramerJeff — APPROVED** [review]

**Comment by KramerJeff** [comment]
Approved, no problems besides React Native being bogus about how they handle bullets

**Pull request merged**